delta/plugins/storage/src/main/resources/query/storages-statistics-aggregations-by-id-proj.json

```diff
@@ -0,0 +1,36 @@
+{
+  "query": {
+    "bool": {
+      "filter": [
+        {
+          "term": {
+            "@type.short": "File"
+          }
+        },
+        {
```

Comment:
I think you can use a single file by adding something like:
```
{{#project}}
{
          "term": {
            "project": "{{project}}"
          }
        }
{{/project}}
```

Reply:
cf. https://scalate.github.io/scalate/documentation/mustache.html#Inverted_Sections